homeassistant/components/transmission/__init__.py

```diff
@@ -447,6 +447,8 @@ def start_torrents(self):
 
     def stop_torrents(self):
         """Stop all active torrents."""
+        if len(self._torrents) == 0:
```

Comment:
Empty containers will be evaluated as falsy.
```py
if not self._torrents:
```

Reply:
The same goes for the `start_torrents` method

Reply:
I can open another PR but not clear on the fix. This?
```python
    def stop_torrents(self):
        """Stop all active torrents."""
        if not self._torrents:
            return
        torrent_ids = [torrent.id for torrent in self._torrents]
        self._api.stop_torrent(torrent_ids)
```

Reply:
Yes, please. :+1: